benchmark_runner/workloads/bootstorm_vm.py

```diff
@@ -66,23 +66,33 @@ def _set_bootstorm_vm_start_time(self, vm_name: str = ''):
         self._bootstorm_start_time[vm_name] = time.time()
 
     @logger_time_stamp
-    def _get_bootstorm_vm_elapsed_time(self, vm_name: str):
+    def _ssh_vm(self, vm_name: str):
+            """
+            Verify ssh into VM and return vm node in success or False if failed
+            @return:
+            """
+            self._virtctl.expose_vm(vm_name=vm_name)
+            # wait till vm ssh login
+            if self._oc.get_vm_node(vm_name=vm_name):
+                vm_node = self._oc.get_vm_node(vm_name=vm_name)
+                if vm_node:
+                    node_ip = self._oc.get_nodes_addresses()[vm_node]
+                    vm_node_port = self._oc.get_exposed_vm_port(vm_name=vm_name)
+                    if self._oc.wait_for_vm_ssh(vm_name=vm_name, node_ip=node_ip, vm_node_port=vm_node_port):
+                        logger.info(f"Successfully ssh into VM: '{vm_name}' in Node: '{vm_node}' ")
+                    return vm_node
+            return False
```

Comment:
Make sure you capture any stderr from the ssh command, so that when it fails you can at least have some hope of figuring out what went wrong (it might simply be a network problem or a credential problem).

Reply:
Does that actually capture the stderr from the ssh command?

Reply:
You can see the full code [here](https://github.com/redhat-performance/benchmark-runner/blob/4e9c29f8977649c9c13659779e87a2095d800c36/benchmark_runner/common/ssh/ssh.py#L24-L41)

Reply:
We have a custom error for it:
raise VMStateTimeout(vm_name=vm_name, state='ssh')

Reply:
But does it report the actual error reported by ssh?  I want that to be reported to make it easier to debug and to distinguish glitches from real problems.  It doesn't look like VMStateTimeout captures the reason for the error (what ssh reports on stderr).